src/UserLogin.ts

```diff
@@ -0,0 +1,21 @@
+import * as consts from "./Consts"
+import getClient from "./Client"
+
+export async function getUser(): Promise<string> {
+  const userIdFromLocalStorage: string | null = localStorage.getItem(
```

Comment:
The compile infers `string | null` here without being told. I think the linter or compiler is going to have an issue with it being stated explicitly when not necessary.

Reply:
Okk ! It will be removed in the next commit.